### PR TITLE
[V2 API] Implement direct observations query from time series cache

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/v2/observation/series.go [hunk 1]
```diff
@@ -17,18 +17,115 @@ package observation
...
+						entityObservation.OrderedFacetObservations = append(
+							entityObservation.OrderedFacetObservations,
+							&pbv2.FacetObservation{
+								FacetId:      facetID,
+								Observations: obsList,
+							},
+						)
+					}
+				}
+			} else if store.MemDb.HasStatVar(variable) {
```
Comment (pradh) [inline internal/server/v2/observation/series.go:93]: If memstore has variable, should that trump what's in KG?

File: internal/server/v2/observation/series.go [hunk 2]
```diff
@@ -17,18 +17,115 @@ package observation
...
+					sort.SliceStable(obsList, func(i, j int) bool {
+						return obsList[i].Date < obsList[j].Date
+					})
+					if queryDate == LATEST {
+						obsList = obsList[len(obsList)-1:]
+					}
+					if len(obsList) > 0 {
+						result.Facets[facetID] = series.Metadata
+						entityObservation.OrderedFacetObservations = append(
+							entityObservation.OrderedFacetObservations,
```
Comment (pradh) [inline internal/server/v2/observation/series.go:118]: If this for loop code is the same maybe can use helper?

**Review by pradh — APPROVED** [review]
Nice!

**Comment by shifucun** [comment]
Thanks for review!

File: internal/server/v2/observation/series.go (hunk 2 above)
Comment (pradh) [reply internal/server/v2/observation/series.go]: That is nice!

**Pull request merged**